cfn_sweeper/base/output.py

```diff
@@ -2,13 +2,15 @@
 import yaml
 from pprint import pprint
 from cfn_sweeper.artwork import Artwork
+
 class ScanReport():
     def __init__(self):
        self.__report_data = {}
 
-    def add_resource_results(self, resource_type:str, managed_resources:list, unmanaged_resources:list):
+    def add_resource_results(self, resource_type:str,stackname:list, managed_resources:list, unmanaged_resources:list):
```

Comment:
why would stack name be a list? Could we not instead take a `dict` for managed_resources and check that it follows the following structure:
```python
{
   'Stack1': [
      'resource1',
      'resource2',
   ],
   'Stack2': [
      'resource1'
   ]
}
```